.github/ISSUE_TEMPLATE/issue--bug-report---feature-request.md

```diff
@@ -1,10 +1,7 @@
 ---
-name: 'Issue: Bug Report / Feature Request'
+name: "Issue: Bug Report / Feature Request"
 about: Create a report to help us improve
-title: ''
+title: ""
 labels: need-to-triage
-assignees: 'shigupt202'
-
+assignees: "kingoliver"
```

Comment:
do you want to make this aksatlanta instead of an individual?

Reply:
Switched to that. Forgot we had a Github team